src/main/java/edu/cornell/kfs/paymentworks/service/impl/PaymentWorksWebServiceImpl.java

```diff
@@ -57,9 +57,7 @@
 
 @Transactional
 public class PaymentWorksWebServiceImpl implements PaymentWorksWebService {
-
-	private static final org.apache.log4j.Logger LOG = org.apache.log4j.Logger
-			.getLogger(PaymentWorksWebServiceImpl.class);
+	private static final org.apache.log4j.Logger LOG = org.apache.log4j.Logger.getLogger(PaymentWorksWebServiceImpl.class);
 
 	private static final String NEW_VENDOR_REQUEST_PENDING_VENDORS = "new-vendor-requests/";
 	private static final String NEW_VENDOR_REQUEST_UPDATE_STATUS = "new-vendor-requests/bulk/";
```

Comment:
I realize this code was not modified with these changes but should directory structures be hard coded this way?  Wouldn't it be better to have these two items in a properties file that allows for easier modification later on?